src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,297 @@
+requested type of asset for a majority of websites and usually take up more bandwidth than any other
+resource. At the 90th percentile, sites send down ~4.7 MB of images on desktop & mobile. That’s a
+lot of cat pictures.
+
+Embedded inline frames use a lot of data too and can also harm page performance. Only loading
+non-critical, below-the-fold images and iframes when the user is likely to see them improves
+page load times, minimize user bandwidth, and reduce memory usage.
+
+Currently, there are two ways to defer the loading of off-screen images and inline frames:
+
+- Using the [Intersection Observer
+  API](https://developers.google.com/web/updates/2016/04/intersectionobserver)
+- Using [`scroll`, `resize`, or `orientationchange` event
+  listeners](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video/#using_event_handlers_the_most_compatible_way)
+
+Either option can let developers include lazy loading functionality, and many have built third-party
+libraries to provide even simpler-to-use abstractions. With lazy loading supported directly by the
+browser however, an external library would not need to be loaded and relied upon. This also ensures
+that deferred loading of images and frames still works even if JavaScript is disabled on the client.
+
+## Lazy loading
+
+Today, Chrome already loads images at different priorities depending on where they are located with
+respect to the device viewport. Although this means that images below the viewport are loaded with a
+lower priority, they are still fetched as soon as possible.
+
+In Chrome 76, the loading attribute can be used to completely defer the loading of offscreen images
+and iframes until the user scrolls near them:
+
+```html
+<img src="image.png" loading="lazy" alt="…" width=”200” height=”200” />
+<iframe src="https://example.com" loading="lazy"></iframe>
+```
+
+The supported values for the attribute are as follows:
+
+- `loading="auto"`: Default lazy-loading behaviour of the browser, which is the same as not
+  including the attribute.
+- `loading="lazy"`: Defer loading of resource until it reaches a certain distance from the viewport.
+- `loading="eager"`: Load resource immediately regardless of where it is located on the page.
+
+Updates to the feature will continue to be rolled in with a final, stable launch in Chrome 76 at the
+earliest, but can currently be tested by enabling the following flags:
+
+- `chrome://flags/#enable-lazy-image-loading`
+- `chrome://flags/#enable-lazy-frame-loading`
+
+### Image loading
+
+In order for images to be lazy loaded effectively, height and width attributes must be included (or
+specified directly in the inline style).
+
+```html
+<img src="…" alt="…" loading="lazy" width="200" height="200"> <!-- lazy-loaded -->
+<img src="…" alt="…" loading="lazy" style="height:200px;width:200px;"> <!-- lazy-loaded -->
+<img src="…" alt="…" loading="lazy"> <!-- not lazy-loaded -->
```

Comment:
TODO: Need to confirm/fix this (images might still be able to load without dimensions but with some reflow)

Reply:
Also need to add a note on console.log that shows for the first image without dimensions ([crbug](https://bugs.chromium.org/p/chromium/issues/detail?id=973124#c2))

Reply:
My intuition is that the console.log implementation won't be ready in time for the post going out. If it is however, +1 to including.